config/jobs/README.md

```diff
@@ -171,6 +171,18 @@ recognize this is going to depend on review latency.
 
 For more details, please refer to [How to Test a ProwJob](/prow/build_test_update.md#how-to-test-a-prowjob)
 
+## Running a Production Job
+
+Normally prow will automatically schedule your job, however if for some reason you
+need to trigger it again you have a few options:
+
+- you can use the rerun feature in prow.k8s.io to run the job again *with the same config*
+- you can use [`config/mkpj.sh`](/config/mkpj.sh) to create a prowjob CR from your local config
+- you can use `bazel run //prow/cmd/mkpj -- --job=foo ...` to create a prowjob CR from your local config
+
+For the latter two options you'll need to submit the resulting CR via `kubectl` configured against
+the prow services cluster.
```

Comment:
I think we need to highlight that this is only available to oncall members. This document is geared toward job authors, most of whom won't have permission to do this.

Reply:
nit: Oncall members or other administrators. One target of these docs are people who run their own Prow instances.

Reply:
not config/, config/ is entirely full of things specifically for prow.k8s.io. this mkpj.sh is for that.

prow/ is supposed to be general but isn't fully yet (mainly prow/cluster/).